R/tidy_sim_results.R

```diff
@@ -0,0 +1,16 @@
+# Function to process simulation results
+tidy_sim_results <- function(sim_data) {
```

Comment:
I'm not 100% sure what this function is doing, but I think it's probably doing this: https://epiverse-trace.github.io/simulist/dev/articles/vis-linelist.html#reshape-line-list-data. If so I don't think you need to write a new function and can apply tidyverse functions over the list of line lists.